[2025-03-28T11:18:03.115+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-28T11:18:03.161+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_3.first_task scheduled__2025-03-27T00:00:00+00:00 [queued]>
[2025-03-28T11:18:03.180+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_3.first_task scheduled__2025-03-27T00:00:00+00:00 [queued]>
[2025-03-28T11:18:03.181+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-03-28T11:18:03.226+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): first_task> on 2025-03-27 00:00:00+00:00
[2025-03-28T11:18:03.242+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=76) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-28T11:18:03.251+0000] {standard_task_runner.py:72} INFO - Started process 82 to run task
[2025-03-28T11:18:03.250+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_3', 'first_task', 'scheduled__2025-03-27T00:00:00+00:00', '--job-id', '128', '--raw', '--subdir', 'DAGS_FOLDER/seconddag.py', '--cfg-path', '/tmp/tmp39_k44bu']
[2025-03-28T11:18:03.253+0000] {standard_task_runner.py:105} INFO - Job 128: Subtask first_task
[2025-03-28T11:18:03.413+0000] {task_command.py:467} INFO - Running <TaskInstance: our_first_dag_3.first_task scheduled__2025-03-27T00:00:00+00:00 [running]> on host 1041ed642999
[2025-03-28T11:18:06.733+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='our_first_dag_3' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2025-03-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-27T00:00:00+00:00'
[2025-03-28T11:18:06.771+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-28T11:18:07.231+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-03-28T11:18:07.255+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', "echo 'hello this first task'"]
[2025-03-28T11:18:07.283+0000] {subprocess.py:99} INFO - Output:
[2025-03-28T11:18:07.287+0000] {subprocess.py:106} INFO - hello this first task
[2025-03-28T11:18:07.289+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-03-28T11:18:07.342+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-28T11:18:07.343+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=our_first_dag_3, task_id=first_task, run_id=scheduled__2025-03-27T00:00:00+00:00, execution_date=20250327T000000, start_date=20250328T111803, end_date=20250328T111807
[2025-03-28T11:18:07.448+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-28T11:18:07.464+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
